installing/disconnected_install/about-installing-oc-mirror-v2.adoc

```diff
@@ -1,39 +1,39 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="about-installing-oc-mirror-v2"]
-= Mirroring images for a disconnected installation using the version 2
+= Mirroring images for a disconnected installation by using the oc-mirror plugin v2
 include::_attributes/common-attributes.adoc[]
 :context: installing-mirroring-disconnected
```

Comment:
I think I've identified why your ifdefs aren't working. This looks to be using the same context as the other oc-mirror assembly. You need to change this to be unique (and it should match the ID)

```suggestion
:context: about-installing-oc-mirror-v2
```

But when you make this change, you need to check ALL other places where you used the `oc-mirror-v2` attribute, to make sure things are rendering not only in v1 and v2 docs, but also there are some shared places with OSD/ROSA docs. Make sure those look correct as well.